edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Http/EdgeHubScopeResult.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.CloudProxy
+{
+    using System.Collections.Generic;
+    using System.Net;
+    using Microsoft.Azure.Devices.Edge.Hub.Http.Controllers;
+    using Newtonsoft.Json;
+
+    public class EdgeHubScopeResult
+    {
```

Comment:
Pls make this immutable

Reply:
the DeviceController is setting the Status, Devices, and Modules separately

---
In reply to: [451646490](https://github.com/Azure/iotedge/pull/3149#discussion_r451646490) [](ancestors = 451646490)